sfcasts/install.md

```diff
@@ -1,61 +1,41 @@
+You can also run `yarn dev-server`, which is another cool way to do this.
+
+Perfect! Then I'm going to open up another tab and say: 
 
-```terminal
+```terminal 
 symfony serve -d
 ```
 
-and that's gonna start a
-local web server using the Symfony binary and it starts at `127.0.0.1:8000`. I'm
-gonna be lazy hold command and click that to say hello to caldron overflow. You've
-been doing our Symfony five series. You're probably familiar with this project, but
-this is not a Symfony 5 project. This is a Symfony 6 project. Ooh, but don't
-worry if you're using Symfony 5, it's not gonna be, it won't be any different.
-
-Now most of the code inside of this project, you don't really need to worry about.
-The most important thing is probably our `src/Entity/` directory. Our doctrine
-entities, our site has questions. Each question has an answer, has a number of
-answers. Each question belongs to a single topic and then we have a user entity. Our
-goal in this tutorial is to create a rich admin section that allows our admin users
-to manage these. So let's get EasyAdmin installed, head terminal and run 
-
-```terminal
+That's going to start a local web server using the Symfony binary and it starts at `127.0.0.1:8000`. I'm gonna be lazy, hold ‘command’ and click that to say “hello” to Cauldron Overflow. If you've been doing our Symfony five series, you're probably familiar with this project, but this is not a Symfony 5 project. This is a Symfony 6 project. But don't worry if you're using Symfony 5. It won't be any different.
+
+Now you don’t really need to worry about most of the code inside this project. The most important thing is probably our `src/Entity/` directory. Our site has questions, and each question has a number of answers. Each question belongs to a single topic and then we have a user entity. Our goal in this tutorial is to create a rich admin section that allows our admin users
+to manage these. So let's get EasyAdmin installed! Find your terminal and run 
+
+```terminal 
 composer require admin
 ```
 
-which is a flex alias, four `easycorp/easyadmin-bundle`. Notice that
-this downloads the brand new version, 4 of EasyAdmin, which only works with
-Symfony 6. So using Symfony 5 run `composer require admin:^3` to
-get version 3 of EasyAdmin bundle right now, version 4 and version 3 are
-identical. So you won't notice any differences, but going forward, new features will
-only be added to version 4. Cool. So this is, is now installed now what? Well, we
-need a dashboard, a starting page for our admin. Let's generate that next.
+which is a flex alias for `easycorp/easyadmin-bundle`. Notice that this downloads the brand new version 4 of EasyAdmin, which only works with Symfony 6. So if you’re using Symfony 5, run `composer require admin:^3` to get version 3 of the EasyAdmin bundle. Right now, version 4 and version 3 are identical, so you won't notice any differences, but going forward, new features will only be added to version 4.
```

Comment:
I try to start a new sentence after a terminal block, instead of continuing a sentence. That's not a HARD rule, but that's how it works most of the time. So, in this case I'd start this paragraph with:

> This is a Flex alias for...

When do we break this rule? Sometimes I say a command *truly* in the middle of a sentence. Something like:

> The best way to run
>
>  ```terminal
> $ git status
> ```
>
> is to find your terminal and type it in.

Most of the time I try not to do this, and we can have a clean new sentence :)

Reply:
Even though I don't type `composer require admin:^3` you can still put that in a `terminal` block. But this is probably a good example where you can continue the sentence after (so it's the "rare" case I was just talking about above!).

Btw, if I ever type a command into the terminal but I don't actually speak it, please put those inside a `terminal` block also - but call it `terminal-silent`. It's nice because it renders like a terminal block on our site, but our subtitle-generating code knows to skip it when making the subtitles.

Reply:
This is probably going to be a dumb question, so I apologize in advance, but how will I know if you don't speak something since this transcript is only your spoken word, right? Will you verbally make note of that in the initial transcript so I'll know to format it like you described above?

I do understand the use of `terminal-silent` though. That makes sense for subtitle purposes.

Reply:
> This is probably going to be a dumb question, so I apologize in advance, but how will I know if you don't speak something since this transcript is only your spoken word, right? Will you verbally make note of that in the initial transcript so I'll know to format it like you described above?

No dumb questions :). These would be... a situation where, in the video, you see me run a command, but that command name doesn't appear on the audio/script. A good example might be if I say:

> Copy that command, spin over to your terminal, and paste.

In the video, I paste some command like `composer require foo/bar`, but I never say that command. That would be a candidate for a `terminal-silent` block. Does that make sense?